docs/samples/Microsoft.ML.Samples.GPU/Microsoft.ML.Samples.GPU.csproj

```diff
@@ -20,9 +20,13 @@
 
   <ItemGroup>
     <ProjectReference Include="..\..\..\src\Microsoft.ML.Vision\Microsoft.ML.Vision.csproj" />
+    <ProjectReference Include="..\..\..\src\Microsoft.ML.ImageAnalytics\Microsoft.ML.ImageAnalytics.csproj" />
+    <ProjectReference Include="..\..\..\src\Microsoft.ML.Data\Microsoft.ML.Data.csproj" />
```

Comment:
Interesting...it seems like we are requiring more project references than before. Any idea why?

Reply:
To get the nuget packages to be built correctly with the correct dependencies, I had to make a lot of the project references be `PrivateAssets`. This makes them not show up as a nuget dependency. The flip side, is that any project that depends on a project with PrivateAssets has to also include the PrivateAssets projects.